test/linting/lint.sh

```diff
@@ -104,10 +98,8 @@ function _shellcheck
     -not -path './test/test_helper/*' \
     -not -path './target/docker-configomat/*'
   )"
-  # macOS lacks parity for `-executable` but presently produces the same results: https://stackoverflow.com/a/4458361
-  [[ "$(uname)" == "Darwin" ]] && FIND_EXEC="-perm -711" || FIND_EXEC="-executable"
   # shellcheck disable=SC2248
-  F_BIN="$(find 'target/bin' ${FIND_EXEC} -type f)"
+  F_BIN="$(grep -l /bin/bash target/bin/*)"
```

Comment:
There are only bash scripts in `target/bin`, with the exception of `acme_extract`. We should make this clear by renaming `acme_extract` to `acme_extract.py`.

Then we can use this:

```suggestion
  F_BIN="$(find 'target/bin' -type f -not -name '*.py')"
```

Reply:
BTW: `print-environment` and `setup` are not executable at the moment and therefore not covered by shellcheck? This should also be fixed with either @NorseGaud or my proposal.

Reply:
> We should make this clear by renaming `acme_extract` to `acme_extract.py`.

Won't that require changing [how it's called then](https://github.com/docker-mailserver/docker-mailserver/blob/d101d0aa0a802eabf74404a8295a0148f98f9497/target/scripts/helpers/ssl.sh#L421)? At least `acme_extract.py` instead of `acme_extract`, or via `python /usr/local/bin/acme_extract.py`?

Reply:
Yes, this:

> acme_extract.py instead of acme_extract